### PR TITLE
Updated services with register_init

### PR DESCRIPTION
**Pull request by Nekseus**

### CODE REVIEW

File: player_service/player_service.lua [hunk 1]
```diff
@@ -20,144 +20,146 @@ do
...
-    end
-
-    if p.update_vitals then
+packets.incoming:register_init({
+    [{0x00D}] = function(p)
+        local data = player.data
+        if p.player_id ~= data.id then
+            return
+        end
+
```
Comment (Nifim) [inline player_service/player_service.lua]: White spaces on line 29,34,39,43,

File: player_service/player_service.lua [hunk 2]
```diff
@@ -20,144 +20,146 @@ do
...
+        local data = player.data
+        if p.player_id ~= data.id then
+            return
+        end
+    
+        if p.update_position then
+            data.movement_speed = p.movement_speed / 8
+            data.animation_speed = p.animation_speed / 8
+        end
+
```
Comment (Nifim) [inline player_service/player_service.lua]: White spaces

File: player_service/player_service.lua [hunk 3]
```diff
@@ -20,144 +20,146 @@ do
...
+        if p.update_position then
+            data.movement_speed = p.movement_speed / 8
+            data.animation_speed = p.animation_speed / 8
+        end
+    
+        if p.update_vitals then
+            data.hp_percent = p.hp_percent
+            data.state = p.state
+        end
+
```
Comment (Nifim) [inline player_service/player_service.lua]: White spaces

File: player_service/player_service.lua [hunk 4]
```diff
@@ -20,144 +20,146 @@ do
...
+    
+        if p.update_vitals then
+            data.hp_percent = p.hp_percent
+            data.state = p.state
+        end
+    
+        if p.update_name then
+            data.name = p.name
+        end
+
```
Comment (Nifim) [inline player_service/player_service.lua]: White spaces on

File: status_effects_service/status_effects_service.lua [hunk 5]
```diff
@@ -15,39 +15,41 @@ status_effects.data = {
...
-        else
-            status_effects.data.player[i] = {
-                id = buff_id,
-                timestamp = (p.durations[i - 1] / 60) + 501079520 + 1009810800 - os.time()
-            }
+packets.incoming:register_init({
+    [{0x063, 9}] = function(p)
+        for i = 1, 0x20 do
+            local buff_id = p.status_effects[i - 1]
+            if buff_id == 0 or buff_id == 0xFF then
```
Comment (Nifim) [inline status_effects_service/status_effects_service.lua]: White spaces after then :) looks like this one was missed in the earlier version.

**Comment by Nekseus** [comment]
Fixed white spaces. I blame VS code!

File: player_service/player_service.lua [hunk 6]
```diff
@@ -20,144 +20,146 @@ do
...
-
-    data.id = p.id
-    data.index = p.index
-    data.hp = p.hp
-    data.mp = p.mp
-    data.tp = p.tp
-    data.hp_percent = p.hp_percent
-    data.mp_percent = p.mp_percent
-end)
+        for i = 0, 0x17 do
```
Comment (z16) [inline player_service/player_service.lua]: The preceeding lines need to be indented to match this indentation.

File: account_service/manifest.xml [hunk 7]
```diff
@@ -1,6 +1,6 @@
 <package>
   <name>account_service</name>
-  <version>1.0.1.0</version>
+  <version>1.0.2.0</version>
```
Comment (z16) [inline account_service/manifest.xml]: You adjusted the `items_service` but increased the version of the `account_service`. Reset this and bump the `items_service` instead.

**Pull request merged**